### PR TITLE
RDM-10064: Updated acr-build.yml to publish service specific images

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-10064

### Change description ###

Service specific Logstash instance

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

**Review by linusnorton — APPROVED** [review]

File: acr-build.yml [hunk 1]
```diff
@@ -2,34 +2,61 @@ version: v1.1.0
...
     push:
       - "{{.Run.Registry}}/ccd/logstash:latest"
-      - "{{.Run.Registry}}/ccd/logstash:ccd-latest"
-      - "{{.Run.Registry}}/ccd/logstash:ccd-{{.Run.ID}}"
-      - "{{.Run.Registry}}/ccd/logstash:probate-latest"
-      - "{{.Run.Registry}}/ccd/logstash:probate-{{.Run.ID}}"
-      - "{{.Run.Registry}}/ccd/logstash:divorce-latest"
-      - "{{.Run.Registry}}/ccd/logstash:divorce-{{.Run.ID}}"
+      - "{{.Run.Registry}}/ccd/logstash:ccd-logstash-latest"
+      - "{{.Run.Registry}}/ccd/logstash:cd-logstash-{{.Run.ID}}"
```
Comment (linusnorton) [inline acr-build.yml:49]: I think you're missing a `c`

File: acr-build.yml [hunk 2]
```diff
@@ -2,34 +2,61 @@ version: v1.1.0
...
+      -t {{.Run.Registry}}/ccd/logstash:ccd-logstash-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-logstash-latest -t {{.Run.Registry}}/ccd/logstash:latest
       --build-arg conf=ccd_logstash.conf.in
       .
     keep: true
 
   - id: probate
     build: >
-      -t {{.Run.Registry}}/ccd/logstash:probate-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:probate-latest
+      -t {{.Run.Registry}}/ccd/logstash:ccd-probate-logstash-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-probate-logstash-latest
       --build-arg conf=ccdprobate_logstash.conf.in
```
Comment (mario-paniccia) [inline acr-build.yml:13]: we might want a similar config for the Probate own instance of LS used to index legacy cases from a Probate own DB, as configured in `probate_logstash.conf.in`
This is  different from the CCD LS that indexes CCD Probate cases from the CCD DB, configured in `ccdprobate_logstash.conf.in`

**Review by mario-paniccia — COMMENTED** [review]
one comment

File: acr-build.yml [hunk 3]
```diff
@@ -2,34 +2,67 @@ version: v1.1.0
 steps:
   - id: ccd
     build: >
-      -t {{.Run.Registry}}/ccd/logstash:ccd-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-latest -t {{.Run.Registry}}/ccd/logstash:latest
+      -t {{.Run.Registry}}/ccd/logstash:ccd-logstash-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-logstash-latest -t {{.Run.Registry}}/ccd/logstash:latest
       --build-arg conf=ccd_logstash.conf.in
       .
     keep: true
 
   - id: probate
```
Comment (mario-paniccia) [inline acr-build.yml:10]:
```suggestion
  - id: ccdprobate
```
see: https://github.com/hmcts/ccd-logstash/blob/b0210df9a3b5b59ad8d5470bf26fe89eade001ec/Jenkinsfile_CNP#L10

File: acr-build.yml (hunk 1 above)
Comment (mario-paniccia) [reply acr-build.yml:53]: can you fix this please

File: acr-build.yml [hunk 4]
```diff
@@ -2,34 +2,67 @@ version: v1.1.0
...
       .
     keep: true
 
+  - id: cmc
+    build: >
+      -t {{.Run.Registry}}/ccd/logstash:ccd-cmc-logstash-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:ccd-cmc-logstash-latest
+      --build-arg conf=ccdcmc_logstash.conf.in
+      .
+    keep: true
+
```
Comment (mario-paniccia) [inline acr-build.yml:34]: can we also add the config for the `sweeper`?

**Review by mario-paniccia — APPROVED** [review]
👍

File: README.md [hunk 5]
```diff
@@ -29,10 +29,10 @@ To create logstash instances of all services CCD is supporting execute build-log
...
- hmcts/ccd/logstash   probate-latest
- hmcts/ccd/logstash   ethos-latest  
- hmcts/ccd/logstash   divorce-latest
- hmcts/ccd/logstash   cmc-latest    
- hmcts/ccd/logstash   ccd-latest  
+ hmcts/ccd/logstash   ccd-sscs-logstash-latest
+ hmcts/ccd/logstash   ccd-probate-logstash-latest
+ hmcts/ccd/logstash   ccd-ethos-logstash-latest
+ hmcts/ccd/logstash   ccd-divorce-logstash-latest
+ hmcts/ccd/logstash   ccd-cmc-logstash-latest
```
Comment (mario-paniccia) [inline README.md:36]: missing the sweeper here?

**Pull request merged**